pools 1 5 1 (#2203)
smartcontractkit/documentation: src/content/ccip/api-reference/v1.5.0/errors.mdx

---
section: ccip
date: Last Modified
title: "CCIP v1.5.0 Error Codes and Messages API Reference"
metadata:
  description: "API documentation for error handling in Chainlink CCIP v1.5.0, including router, onramp, rate limiter, and token-related errors."
  canonical: "/ccip/api-reference/v1.5.1/errors"
---

import CcipCommon from "@features/ccip/CcipCommon.astro"
import { CCIPSendError } from "@features/ccip/components/api-reference"
import { Aside, CopyText } from "@components"

<CcipCommon callout="importCCIPPackage150" />

> When invoking the `ccipSend` [function](/ccip/api-reference/v1.5.0/i-router-client#ccipsend), it is possible to encounter various errors. These might be thrown either by the CCIP router or by one of the downstream contracts called by the CCIP router. Below is a compiled list of potential errors you might encounter. Referencing this list will enable you to capture and handle these exceptions gracefully.

Reviewer: ```suggestion
If you get errors when invoking the `ccipSend` [function](/ccip/api-reference/v1.5.0/i-router-client#ccipsend), the errors might be thrown either by the CCIP router or by one of the downstream contracts called by the CCIP router. Below is a compiled list of potential errors you might encounter. Referencing this list will enable you to capture and handle these exceptions gracefully.
```
